product_manager.py

```diff
@@ -8,35 +8,35 @@ def __init__(self, item_manager, tag_manager, image_manager):
         self.tag_manager = tag_manager
         self.image_manager = image_manager
 
-    def add_product(self, tags, vector, png):
+    def add_product(self, tags, vector_file, png_file):
         item_id = helper.get_id()
-        vector_path = self.image_manager.add_image(vector, item_id, True)
-        png_path = self.image_manager.add_image(png, item_id, False)
+        vector_path = self.image_manager.add_vector_image(vector_file, item_id)
+        png_path = self.image_manager.add_png_image(png_file, item_id)
         self.item_manager.add_item(item_id, vector_path, png_path)
         for tag in tags:
             self.tag_manager.add_tag(tag, item_id)
+        return item_id
 
     def delete_product(self, item_id):
         self.tag_manager.delete_tags_for_item(item_id)
         self.item_manager.delete_item(item_id)
         self.image_manager.delete_image(item_id)
-    
+
     def get_all_products(self):
         all_items = self.item_manager.get_all_items()
         for item in all_items:
             item_id = item["ItemId"]
             item_tag_list = self.tag_manager.get_item_tags(item_id)
-            item.update({'Tags' : item_tag_list})
```

Comment:
No need for .update, isn't `item['Tags'] = item_tag_list` enough?

Reply:
already updated as per previous comment, think this is the old code